docs/core/tools/dotnet-sln.md

```diff
@@ -31,42 +31,64 @@ dotnet new sln
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
 
 ## Options
 
 - **`-h|--help`**
 
-  Prints out a short help for the command.
+  Prints options for the command.
 
 ## Commands
 
+### `list`
+
+Lists all projects in a solution file.
+
+#### Synopsis
+
+```dotnetcli
+dotnet sln list [-h|--help]
+```
+  
+#### Arguments
+
+- **`SOLUTION_FILE`**
+
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
```

Comment:
In the H4 Arguments sections I would omit the description for SOLUTION_FILE since that's covered at the top in the H2 Arguments section. If we keep the H4 descriptions we should make sure they're all the same.

Reply:
I'd prefer to keep them, since I can easily imagine people skipping down to what they think is relevant, then getting confused. I made this one match the one above, though.